File: exampleSources/scala/app1.scala
// Scala — case class + pattern match + sort.

case class Task(title: String, priority: Int, done: Boolean)

object Main extends App {
  val backlog = List(
    Task("Write tests",     2, false),
    Task("Fix login bug",   5, false),
    Task("Refactor parser", 3, true),
    Task("Ship release",    1, false),
  )

  val open = backlog.filter(!_.done)
  val ordered = open.sortBy(_.priority)
  val titles = ordered.map(_.title)

  titles.foreach(println)
}

File: exampleSources/swift/app1.swift
// Swift — struct + map + sort.

struct Task: Equatable {
    var title: String
    var priority: Int
    var done: Bool
}

let backlog: [Task] = [
    .init(title: "Write tests",     priority: 2, done: false),
    .init(title: "Fix login bug",   priority: 5, done: false),
    .init(title: "Refactor parser", priority: 3, done: true),
    .init(title: "Ship release",    priority: 1, done: false),
]

let open = backlog.filter { !$0.done }
let ordered = open.sorted { $0.priority < $1.priority }
let titles = ordered.map(\.title)

for t in titles {
    print(t)
}